test/money-market/bToken.test.ts

```diff
@@ -0,0 +1,99 @@
+    bTokenAdmin = bTokens.bTokenAdmin;
+
+    usdc = <ERC20>await ethers.getContractAt('ERC20', await bUSDC.underlying());
+    weth = <ERC20>await ethers.getContractAt('ERC20', await bWETH.underlying());
+    await weth.transfer(softVault.address, await weth.balanceOf(admin.address));
+  });
+
+  describe('Mint', () => {
+    it('should revert when caller is not soft vault', async () => {
+      const amount = ethers.BigNumber.from(ethers.utils.randomBytes(32));
+      await expect(bUSDC.mint(amount)).to.be.revertedWith('caller should be softvault');
+      await expect(bWETH.mint(amount)).to.be.revertedWith('caller should be softvault');
+    });
+
+    it('should not revert when caller is soft vault', async () => {
+      const amount = ethers.BigNumber.from(ethers.utils.randomBytes(2));
+      await bTokenAdmin._setSoftVault(bUSDC.address, softVault.address);
+      await bTokenAdmin._setSoftVault(bWETH.address, softVault.address);
+
+      await expect(bUSDC.connect(softVault).mint(amount)).to.not.be.revertedWith('caller should be softvault');
+      await expect(bWETH.connect(softVault).mint(amount)).to.not.be.revertedWith('caller should be softvault');
+
+      await usdc.connect(softVault).approve(bUSDC.address, amount);
+      let returnValue = await bUSDC.connect(softVault).callStatic.mint(amount);
+      expect(returnValue).to.equal(0);
+
+      await weth.connect(softVault).approve(bWETH.address, amount);
+      returnValue = await bWETH.connect(softVault).callStatic.mint(amount);
+      expect(returnValue).to.equal(0);
+    });
+  });
+
+  describe('Borrow', () => {
+    it('should revert when caller is not bank', async () => {
+      const amount = ethers.BigNumber.from(ethers.utils.randomBytes(32));
+      await expect(bUSDC.borrow(amount)).to.be.revertedWith('only bank can borrow');
+      await expect(bWETH.borrow(amount)).to.be.revertedWith('only bank can borrow');
+    });
+
+    it('should not revert when caller is bank', async () => {
+      let amount = ethers.BigNumber.from(ethers.utils.randomBytes(2));
+      await comptroller._setCreditLimit(bank.address, bUSDC.address, amount);
+      await comptroller._setCreditLimit(bank.address, bWETH.address, amount);
+
+      amount = ethers.BigNumber.from(10);
+
+      await expect(bUSDC.connect(bank).borrow(amount)).to.not.be.revertedWith('only bank can borrow');
+      await expect(bWETH.connect(bank).borrow(amount)).to.not.be.revertedWith('only bank can borrow');
+
+      await usdc.connect(softVault).approve(bUSDC.address, amount);
+      await bUSDC.connect(softVault).mint(amount);
+
+      let returnValue = await bUSDC.connect(bank).callStatic.borrow(amount.div(2));
+      expect(returnValue).to.equal(0);
+
+      await weth.connect(softVault).approve(bWETH.address, amount);
+      await bWETH.connect(softVault).mint(amount);
+
+      returnValue = await bWETH.connect(bank).callStatic.borrow(amount.div(2));
+      expect(returnValue).to.equal(0);
```

Comment:
```suggestion
      let success = await bUSDC.connect(bank).callStatic.borrow(amount.div(2));
      expect(success).to.equal(0);

      await weth.connect(softVault).approve(bWETH.address, amount);
      await bWETH.connect(softVault).mint(amount);

      success = await bWETH.connect(bank).callStatic.borrow(amount.div(2));
      expect(success).to.equal(0);
```

Rename return value to improve clarity